operators/samples/templates.yaml

```diff
@@ -51,3 +56,79 @@ spec:
   workspace.crownlabs.polito.it/WorkspaceRef:
     name: coffee
   deleteAfter: 1h
+---
+apiVersion: crownlabs.polito.it/v1alpha2
+kind: Template
+metadata:
+  name: vscode-python-proxyenabled
+  namespace: workspace-standalone
+spec:
+  prettyName: Visual Studio Code - Python (proxy enabled)
+  description: A template about vscode
+  environmentList:
+    - name: vscode-environment
+      environmentType: Standalone
+      mode: Standard
+      image: harbor.crownlabs.polito.it/crownlabs-standalone/vscode-python:latest
```

Comment:
Never, ever use the `latest` tag.

Reply:
Ok, i've changed also the templates that have been uploaded

Reply:
Thanks

Reply:
Ehm... If you do not put any tags it is the same as latest

Reply:
Lapsus